alibi/explainers/anchor_image.py

```diff
@@ -247,7 +246,7 @@ def perturbation(
             mask = np.zeros(segments.shape).astype(bool)
             for superpixel in to_perturb:
                 mask[segments == superpixel] = True
-            if background_idx:
+            if background_idx is not None:
```

Comment:
The main bug. Here `background_idx` can be `0` but in the previous version it would cause it to go down this (wrong) branch and end up using `fudged_image` which is not defined. The fix is to explicitly check for `None`.